examples/fluids/blasius.yaml

```diff
@@ -21,14 +21,15 @@ stab: 'supg'
 Ctau_t: 1
 #Ctau_v: 36,60,128 is what PHASTA has for p=1,2, 3
 Ctau_v: 36
-Ctau_C: 0.125
-Ctau_M: 1.0
-Ctau_E: 0.125
+Ctau_C: 0.5
+Ctau_M: 0.5
+Ctau_E: 0.5
```

Comment:
Are the old ones appropriate for quadratic?
```suggestion
Ctau_C: 0.5
Ctau_M: 0.5
Ctau_E: 0.5
## Quadratic settings
#Ctau_v: 60
#Ctau_C: 0.125
#Ctau_M: 0.125
#Ctau_E: 0.125
```

Reply:
I am running some tests on this as I am unsure.  What I am observing so far is better convergence for large time steps  with 0.5 than reduced values.  I need to dig into the docs to figure out how to force SNES to always take two iterations as it is converging well until SNES decides it needs only on iteration at which time one iteration and then on the next step it fails to converge.  Obviously this is just SNES settings but it needs to be worked out if we want to push large time steps.

Reply:
I think that's happening once already very close to steady and the issue will go away with analytic Jacobian.